jupyterhub/alembic/env.py

```diff
@@ -34,7 +34,8 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-target_metadata = None
+from jupyterhub import orm
```

Comment:
Do we still want to support the comments in line 33-36? If so, I would combine lines 33 & 34 for greater clarity.

Reply:
Done.